dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/action/ViewCMSMaintenanceAction.java

```diff
@@ -885,17 +873,13 @@ else if(Identifier.class.equals(clazz)){
 			
 			file = new File(backupTempFilePath + "/index_live.json");
 			new ESIndexAPI().backupIndex(info.live, file);
-
-
-			
-			
-			
-			
 			
 		} catch (HibernateException e) {
 			Logger.error(this,e.getMessage(),e);
 		} catch (SystemException e) {
 			Logger.error(this,e.getMessage(),e);
+		} catch (Exception e) {

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Combine this catch with the one at line 877, which has the same body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2147)